src/api.py

```diff
@@ -155,6 +169,7 @@ def _decode(self, text, decode_type):
                 22, "Decode error - %s. Had trouble while decoding string: %r"
                 % (e, text)
             )
+        return None
```

Comment:
Any function without `return` statement returns `None` by default

Reply:
I know this. Pylint want it explicit here.